src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -78,6 +84,11 @@ public Path getHrManagerPositionsFilePath() {
         return model.getHrManagerPositionsFilePath();
     }
 
+    @Override
+    public Path getHrManagerInterviewFilPath() {
+        return model.getHrManagerInterviewsFilePath();
```

Comment:
Misspelling here, should be getHrManagerInterviewFilePath()

Reply:
Ah I see the error there. The misspelling should be at the method signature. Btw I think I will put it as `getHrManagerInterviewsFilePath()`, to make it consistent with `getHrManagerPositionsFilePath()` and `getHrManagerCandidatesFilePath()`. Thx a lot for pointing that out.